.gitignore

```diff
@@ -39,14 +39,14 @@ playground.xcworkspace
 # Swift Package Manager
 #
 # Add this line if you want to avoid checking in source code from Swift Package Manager dependencies.
-# Packages/
-# Package.pins
-# Package.resolved
-# *.xcodeproj
-#
+Packages/
+Package.pins
+Package.resolved
+*.xcodeproj
```

Comment:
I think we actually need this to stay since we have the Tools App project in here

Reply:
Ok re-added them, except `Package.resolved` and `.pins` which are generated by SPM so don't need to be tracked in this case.